render/image.go

```diff
@@ -147,12 +159,66 @@ func (p *Image) InitFromImage(data []byte) error {
 	return nil
 }
 
-func (p *Image) Init() error {
-	err := p.InitFromWebP([]byte(p.Src))
+func (p *Image) InitFromSVG(data []byte) error {
+	svgData, _ := oksvg.ReadIconStream(bytes.NewReader(data), oksvg.StrictErrorMode)
+	w := int(svgData.ViewBox.W)
+	h := int(svgData.ViewBox.H)
+
+	if w == 0 && h == 0 {
+		return errors.New("decoding svg data failed")
+	}
+
+	svgData.SetTarget(0, 0, float64(w), float64(h))
+	rgba := image.NewRGBA(image.Rect(0, 0, w, h))
+	svgData.Draw(rasterx.NewDasher(w, h, rasterx.NewScannerGV(w, h, rgba, rgba.Bounds())), 1)
+	buf := new(bytes.Buffer)
+	err := jpeg.Encode(buf, rgba, nil)
+
 	if err != nil {
-		err = p.InitFromGIF([]byte(p.Src))
+		return err
+	}
+
+	err = p.InitFromImage([]byte(buf.Bytes()))
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (p *Image) InitFromURL(url string) error {
```

Comment:
Ah I had a hunch this wasn't added originally because of caching concerns! Thank you for the insight into that process, Ill definitely update the PR to remove this feature 👍

Reply:
Removed!